[2024-09-22T15:13:31.474+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T15:13:31.530+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NY_TAXI.download_data manual__2024-09-22T15:13:25.566220+00:00 [queued]>
[2024-09-22T15:13:31.563+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NY_TAXI.download_data manual__2024-09-22T15:13:25.566220+00:00 [queued]>
[2024-09-22T15:13:31.564+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-22T15:13:32.905+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): download_data> on 2024-09-22 15:13:25.566220+00:00
[2024-09-22T15:13:32.922+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'NY_TAXI', 'download_data', 'manual__2024-09-22T15:13:25.566220+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/download.py', '--cfg-path', '/tmp/tmpqzr5azrp']
[2024-09-22T15:13:32.926+0000] {standard_task_runner.py:105} INFO - Job 2: Subtask download_data
[2024-09-22T15:13:37.245+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=311) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-22T15:13:37.246+0000] {standard_task_runner.py:72} INFO - Started process 314 to run task
[2024-09-22T15:13:37.875+0000] {task_command.py:467} INFO - Running <TaskInstance: NY_TAXI.download_data manual__2024-09-22T15:13:25.566220+00:00 [running]> on host 8c9160901df5
[2024-09-22T15:13:44.173+0000] {job.py:229} INFO - Heartbeat recovered after 15.01 seconds
[2024-09-22T15:13:48.075+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='NY_TAXI' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T15:13:25.566220+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T15:13:25.566220+00:00'
[2024-09-22T15:13:48.091+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T15:13:54.682+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-22T15:13:55.137+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2024-09-22']
[2024-09-22T15:13:57.276+0000] {subprocess.py:86} INFO - Output:
[2024-09-22T15:13:59.175+0000] {subprocess.py:93} INFO - 2024-09-22
[2024-09-22T15:13:59.711+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-22T15:14:01.385+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-22T15:14:01.386+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=NY_TAXI, task_id=download_data, run_id=manual__2024-09-22T15:13:25.566220+00:00, execution_date=20240922T151325, start_date=20240922T151331, end_date=20240922T151401
[2024-09-22T15:14:02.387+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-22T15:14:02.802+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-22T15:14:02.954+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
